src/chrome/content/rules/watson.ch.xml

```diff
@@ -1,6 +1,44 @@
+<!--
+	Cert mismatch:
+		- won.chww.watson.ch
+		- rts.chwww.watson.ch
+		- www.sorrynopenadanixzuberichten.watson.ch
+-->
 <ruleset name="Watson.ch">
 	<target host="watson.ch"/>
 	<target host="www.watson.ch"/>
 
+	<target host="8h3b0rvt4w.watson.ch"/>
+	<target host="blwww.watson.ch"/>
+	<target host="cdn1.watson.ch"/>
+	<target host="cdn2.watson.ch"/>
+	<target host="cdn3.watson.ch"/>
+	<target host="cdn4.watson.ch"/>
+	<target host="cdn5.watson.ch"/>
+	<target host="cdn6.watson.ch"/>
+	<target host="dns1.watson.ch"/>
+	<target host="dns2.watson.ch"/>
+	<target host="dns3.watson.ch"/>
+	<target host="dns4.watson.ch"/>
+	<target host="gowww.watson.ch"/>
+	<target host="hipsterlitheater.watson.ch"/>
+	<target host="http.watson.ch"/>
+	<target host="hwww.watson.ch"/>
+	<target host="iwww.watson.ch"/>
+	<target host="job.watson.ch"/>
+	<target host="jobs.watson.ch"/>
+	<target host="ntdw5ophfx.watson.ch"/>
+	<target host="parabolawww.watson.ch"/>
+	<target host="pkiwl1cxgy.watson.ch"/>
+	<target host="q3wcxptmgv.watson.ch"/>
+	<target host="v8eupxtsbj.watson.ch"/>
+	<target host="w.watson.ch"/>
+	<target host="w10ziy8aft.watson.ch"/>
+	<target host="ww.watson.ch"/>
+	<target host="wwww.watson.ch"/>
+	<target host="yakbs46r0p.watson.ch"/>
+
 	<rule from="^http:" to="https:" />
 </ruleset>
+
+
```

Comment:
They use a wildcard DNS and cert, so listing subdomains makes not really sense here.

Reply:
not all of them , i will recheck.
CDN  domains are used though.